k8s/interpreter/100-interpreter-pod.yaml

```diff
@@ -0,0 +1,143 @@
+#
```

Comment:
The name of this file mentions "pod". But the yaml contains other resources like svc, rbac. Probably a better name can be used like 100-interpreter-spec.yaml?

Reply:
File renamed.